src/usb-import

```diff
@@ -1,6 +1,7 @@
-#!/bin/sh
+#!/bin/sh --
 
-command -v modprobe >/dev/null && (modprobe vhci-hcd || exit 1)
+if command -v modprobe >/dev/null; then modprobe vhci-hcd || exit 1; fi
+set -eu
```

Comment:
Same comment as to the other script.

Reply:
I checked with this version of the script and it worked on my system.